test/WebJobs.Script.Tests/Description/Node/NodeFunctionInvokerTests.cs

```diff
@@ -1,6 +1,7 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
+#if NODE
```

Comment:
What is this?

Reply:
Tests I haven't yet ported, it's been mostly integration thus far.

Reply:
This was a symbol added when first migrating to disable Node (as it wasn't supported). Now that the Node invoker is gone, I'll remove this.

Reply:
There might still be some valuable test coverage - I want to take a look at it before removing.